velox/exec/StreamingAggregation.cpp

```diff
@@ -276,14 +313,23 @@ RowVectorPtr StreamingAggregation::getOutput() {
   std::iota(newGroups.begin(), newGroups.end(), numPrevGroups);
 
   for (auto i = 0; i < aggregates_.size(); ++i) {
-    auto& aggregate = aggregates_.at(i).function;
+    auto& function = aggregates_.at(i).function;
 
-    aggregate->initializeNewGroups(
+    function->initializeNewGroups(
         groups_.data(), folly::Range(newGroups.data(), newGroups.size()));
   }
 
   evaluateAggregates();
 
+  // Initialize new groups and evaluate aggregates in sortedAggregations.
```

Comment:
Shouldn't we skip sorted aggs in the loop above and in evaluateAggregates?

Reply:
Fixed

Reply:
Somehow I don't see logic to skip sorted aggs in the loop above. See https://github.com/facebookincubator/velox/pull/7764 for context.

Reply:
Fixed, and move the sortedAggrgation eval into the evaluateAggregates method.